DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/SectionContentViewModelTests.cs

```diff
@@ -532,108 +402,104 @@ public void Diagnostic_assessment_completion_status_does_not_use_plural_if_attem
             sectionContentViewModel.DiagnosticCompletionStatus.Should().Be($"{secScore}/{secOutOf} - {diagAttempts} attempt");
         }
 
-        [Test]
-        public void Diagnostic_assessment_separator_is_false_if_no_diagnostic_assessment()
+        [TestCase(true, true, true, true, true)]
+        [TestCase(true, false, true, true, true)]
+        [TestCase(true, false, false, true, true)]
+        [TestCase(true, false, false, false, false)]
+        [TestCase(false, true, true, true, false)]
```

Comment:
I think you're missing some cases here. We want to test all the permutations of the four booleans, which is why a parameterised test is so useful 🙂